core/src/test/scala/neotypes/StreamSuite.scala

```diff
@@ -30,6 +31,8 @@ abstract class BaseStreamSpec[S[_], F[_]](streamTestkit: StreamTestkit[S, F]) ex
 
   protected implicit final val S: Stream.Aux[S, F] =
     behaviour.streamInstance
+
+  protected final def streamConcurrently[A](stream1: S[Unit], stream2: S[Unit]) = behaviour.streamConcurrently(stream1, stream2)
```

Comment:
Please add an explicit return type to this method and move the implementation to the next line.